src/nadzoring/plugins/connectors/dns.py

```
@@ -0,0 +1,401 @@
+        tags=(*_TAGS, "whois"),
+    )
+
+    domain: str
+
+    def probe(self) -> ProbeResult:
+        from nadzoring.network_base.whois_lookup import whois_domain_lookup
+
+        try:
+            data = whois_domain_lookup(self.domain)
+            return _ok(data)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)